ext/cfx-ui/src/cfx/apps/mpMenu/parts/LegacyConnectingModal/AdaptiveCardPresenter/AdaptiveCardPresenter.module.scss

```diff
@@ -185,6 +186,13 @@
           border-color: ui.color('error');
         }
       }
+
+      &:global(.ac-pushButton-disabled) {
+        color: ui.color-token('button-disabled-text');
+        border-color: ui.color-token('button-disabled-border');
+        background-color: ui.color-token('button-disabled-background');
+        cursor: default;
```

Comment:
Please use the same style here as we already have for disabled buttons: https://github.com/citizenfx/fivem/blob/master/ext/cfx-ui/src/cfx/ui/Button/Button.module.scss#L67-L70